### DISCUSSION TITLE
Details about CH4 example in tutorial

### DISCUSSION THREAD

**Discussion by XiaMinqi** [deepmodeling/dpgen#1088] [Q&A] [answered]
Hi developers, I encountered some problems when testing the CH4 example provided in tutorial. 

The accurate rate in inter.000001 didn't reach 100% as expected (due to the random seed perhaps), the result is: 
```
INFO:dpgen:system 001 candidate :   1625 in   3010  53.99 %
INFO:dpgen:system 001 failed    :    830 in   3010  27.57 %
INFO:dpgen:system 001 accurate  :    555 in   3010  18.44 %
```
Thus the model is far from ideal and it should be trained in more iterations. 
I noticed that in param.json, for each iteration there is an indeopendent group of structures to be explored (000000 ~ 000009 for iter000000, 000010 ~ 000019 for iter000001)
If I'm going to perform more iterations, is it necessary to add more structure files in “sys_configs” or can I just use the same group of structures for model_devi in different iterations until acuurate rate reaches 100%?

**Comment by AnguseZhang** [comment, marked as answer]
Is the accurate ratio of iter1 higher than iter0?  If it's true, this is normal. Since this is just one demo tutorial, we do not expect it reaches 100%. 

Meanwhile as you say, you can try to perform some other iterations to make the model converged.

**Reply by XiaMinqi** [reply]
Yes, accurate ratio is usually higher in latter iterations. Thank you for your kind reply!
